sql/table.sql

```diff
@@ -1999,7 +2009,7 @@ CREATE TABLE stable_id_event (
   new_stable_id             VARCHAR(128),
   new_version               SMALLINT,
   mapping_session_id        INT(10) UNSIGNED NOT NULL DEFAULT '0',
-  type                      ENUM('gene', 'transcript', 'translation') NOT NULL,
+  type                      ENUM('gene', 'transcript', 'translation', 'rnaproduct') NOT NULL,
```

Comment:
Need to add rnaproduct to the column type documentation above

Reply:
Well spotted. Fixed.